components/tooltip/tooltip.js

```diff
@@ -976,6 +976,7 @@ class Tooltip extends RtlMixin(LitElement) {
 		const clone = target.cloneNode(true);
 		clone.removeAttribute('id');
 		clone.style.maxWidth = 'none';
+		clone.style.display = 'inline-block';
```

Comment:
This is a bit worrying, but without this the `clone.scrollWidth` is not properly calculated. `show-truncated-only` is only used in a couple places outside `core` so I can take a look. If this is an issue, we could also only apply it if the target is a link.

Reply:
Ok, consistent-eval and outcomes are both still working fine with this.  Also reproduced the setup Nova has and it works as well.  Opened an optional PR to add that use case here, if we want: https://github.com/BrightspaceUI/core/pull/3848